tests/cromwellapi/validate_wdls.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from utils import fetch_wdl_paths
+
+logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+
+
+def validate_wdls(wdl_paths):
+    [check_wdl_name_matches(w) for w in wdl_paths]
+    [check_for_other_json_files(w) for w in wdl_paths]
+
+
+def check_wdl_name_matches(path):
+    # print(f"Checking {path}")
+    if not path.name.replace(".wdl", "") == path.parent.name:
+        logging.info(
+            f"WDL file name {path.name} and dir name {path.parent.name} do not match"
+        )
+
+
+def check_for_other_json_files(path):
+    jsons = list(path.parent.glob("*.json"))
+    if jsons:
+        for json in jsons:
+            if json.name not in ["options.json", "inputs.json"]:
+                logging.info(
```

Comment:
Instead of info() can we use warning() so it shows up as WARNING in the output?

Reply:
@seankross just for this check or do you mean for all checks?

Reply:
![Screenshot 2025-05-19 at 10 19 02 AM](https://github.com/user-attachments/assets/40086cbf-f9a2-4531-ba99-d78e70a755f6)

@seankross changed to warnings for both checks. using `rich` for colored logging and it automatically shows line numbers